definitions/firefox_ios.toml

```diff
@@ -250,7 +250,7 @@ description = "Number of times Contile Sponsored Tiles setting is flipped."
 owner = "xluo-all@mozilla.com"
 
 [metrics.new_profile_activation]
-select_expression = "COALESCE(SUM(activated))"
+select_expression = "COUNTIF(is_activated)"
```

Comment:
@danielkberry @mikewilli  should we create a new metric here, or change the existing one in place?